src/components/atoms/ImageInput/ImageInput.tsx

```diff
@@ -97,11 +101,37 @@ export const ImageInput: React.FC<ImageInputProps> = ({
 
   const labelClasses = classNames("ImageInput__container", {
     "ImageInput__container--error": !!error?.message,
-    "ImageInput__container--small": small,
     "ImageInput__container--disabled": loading,
     "mod--hidden": isInputHidden || !imageUrl,
   });
 
+  console.log("imgUrl", fileUrl);
+
+  const wrapperClasses = classNames(
+    "ImageInput__wrapper justify-center items-center flex bg-cover",
+    {
+      "w-full text-white rounded-md": variant === "wide",
+      "w-48 h-48 text-white rounded-full flex-shrink-0 mx-auto":
+        variant === "round",
+    }
+  );
+
+  const wrapperStyles = useMemo(
+    () =>
+      variant === "wide"
+        ? {
+            height: "150px",
+            backgroundImage: `url(${imgUrl || DEFAULT_IMAGE_INPUT_BACKGROUND})`,
+          }
+        : {
+            backgroundImage: `url(${
+              imgUrl || DEFAULT_IMAGE_INPUT_ROUND_BACKGOUND
+            })`,
+            backgroundPosition: "100% 100%",
+          },
+    [imgUrl, variant]
+  );
```

Comment:
[NOTE] This looks like an overkill. There might be a way to split it into two-three simpler child components, but if it works like this, it's good enough.

Reply:
added .tailwind.ts module and tried to simplify the `wrapperStyles` object.
I tried to add a variant for 2 image inputs: wide and round (see the screenshot)
<img width="466" alt="image" src="https://user-images.githubusercontent.com/63313732/155709852-7d1aced4-176f-43df-a5c5-45afaad2f5cd.png">

they both serve image uploading and have different default background, that's why the styles differ a bit as well.